pkg/liqoctl/status/remoteinfo_test.go

```diff
@@ -0,0 +1,272 @@
+	var (
+		clientBuilder fake.ClientBuilder
+		rootNode      = newRootInfoNode(rootTitle)
+		ric           *RemoteInfoChecker
+	)
+
+	BeforeEach(func() {
+		ctx = context.Background()
+		_ = netv1.AddToScheme(scheme.Scheme)
+		clientBuilder = *fake.NewClientBuilder().WithScheme(scheme.Scheme)
+	})
+
+	Context("Creating a new remoteInfoChecker", func() {
+		JustBeforeEach(func() {
+			ric = newRemoteInfoChecker(namespace, &[]string{}, &[]string{}, clientBuilder.Build())
+		})
+		It("should return a valid remoteInfoChecker", func() {
+			ricTest := &RemoteInfoChecker{
+				client:             clientBuilder.Build(),
+				namespace:          namespace,
+				clusterNameFilter:  &[]string{},
+				clusterIDFilter:    &[]string{},
+				errors:             false,
+				collectionErrors:   nil,
+				rootRemoteInfoNode: rootNode,
+			}
+			Expect(*ric).To(Equal(*ricTest))
+		})
+	})
+
+	Context("Checking if a cluster ID or a cluster name are included between filters", func() {
+		Context("Checking cluster ID", func() {
+			BeforeEach(func() {
+				ric.clusterIDFilter = &[]string{"cluster-id-1", "cluster-id-2"}
+			})
+			When("Cluster ID is included", func() {
+				It("should return true", func() {
+					Expect(ric.argsFilterCheck("cluster-id-1", "")).To(BeTrue())
+				})
+			})
+			When("Cluster ID is not included", func() {
+				It("should return false", func() {
+					Expect(ric.argsFilterCheck("cluster-id-3", "")).To(BeFalse())
+				})
+
+			})
+		})
+		Context("Checking cluster name", func() {
+			BeforeEach(func() {
+				ric.clusterNameFilter = &[]string{"cluster-name-1", "cluster-name-2"}
+			})
+			When("Cluster name is included", func() {
+				It("should return true", func() {
+					Expect(ric.argsFilterCheck("", "cluster-name-1")).To(BeTrue())
+				})
+			})
+			When("Cluster name is not included", func() {
+				It("should return false", func() {
+					Expect(ric.argsFilterCheck("", "cluster-name-3")).To(BeFalse())
+				})
+
+			})
+		})
+
+	})
+
+	Context("Getting Collect() result", func() {
+		var infoNode InfoNode
+		BeforeEach(func() {
+			clientBuilder.WithObjects(
+				&v1.ConfigMap{
+					TypeMeta: metav1.TypeMeta{
+						Kind: "ConfigMap",
+					},
+					ObjectMeta: metav1.ObjectMeta{
+						Labels: map[string]string{
+							"app.kubernetes.io/name": "clusterid-configmap",
+						},
+						Namespace: namespace,
+					},
+					Immutable: new(bool),
+					Data: map[string]string{
+						"CLUSTER_ID":   clusterID1,
+						"CLUSTER_NAME": clusterName1,
+					},
+					BinaryData: map[string][]byte{},
+				},
+				&netv1.NetworkConfig{
+					TypeMeta: metav1.TypeMeta{
+						Kind: "NetworkConfig",
+					},
+					ObjectMeta: metav1.ObjectMeta{
+						Name: clusterName1,
+						Labels: map[string]string{
+							"liqo.io/originID":    clusterID2,
+							"liqo.io/remoteID":    clusterID1,
+							"liqo.io/replication": "false",
+						},
+						OwnerReferences: []metav1.OwnerReference{
+							{
+								Name: clusterName2,
+							},
+						},
```

Comment:
```suggestion
						OwnerReferences: []metav1.OwnerReference{{Name: clusterName2}},
```

Avoid wasting unnecessary space when possible.